app/viewModels/DepartureStatus.scala

```diff
@@ -49,6 +51,9 @@ object DepartureStatus {
       DepartureStatus("departure.status.mrnAllocated", Nil)
   }
 
+  private def viewGuaranteeValidationFailAction(departureId: DepartureId, config: FrontendAppConfig) =
+    ViewMovementAction(config.departureFrontendRejectedUrl(departureId), "viewDepartures.table.action.viewErrors")
```

Comment:
Could we not call controller directly like downloadTADAction method? This would allow you to get rid of the config argument (the link is within the same service so calling the controller with departure id would be fine)